src/Microsoft.DotNet.Build.Tasks.Packaging/test.Desktop/project.json

```diff
@@ -8,9 +8,11 @@
     "xunit.runner.visualstudio": "2.1.0"
   },
   "frameworks": {
-    "net45": { }
```

Comment:
I believe that this tests do not even run when we do a build of buildtools, which is why it didn't need it. @ericstj is there a reason why Desktop packaging tests do not run on builds?